cterasdk/core/plans.py

```diff
@@ -9,10 +12,21 @@ class Plans(BaseCommand):
     """
     default = ['name']
 
-    def get(self, name, include=None):
+    def get(self, name):
```

Comment:
Is it intended for users to call this method or just for internal use?

Reply:
I think we need a 'get' command for any object on the Portal. In many scenario's, you cannot update individual attributes and you're forced to submit the entire object back. A 'get' function allows us to retrieve the entire object, make changes and submit it back